### DISCUSSION TITLE
How to add or modify the quicklist of central wavelength for Sentinel-2

### DISCUSSION THREAD

**Discussion by FranklinJS** [semiautomaticgit/SemiAutomaticClassificationPlugin#159] [Q&A] [answered]
How to add or modify the quicklist of central wavelength for Sentinel-2 to conform to table 1 (10 m) for S2A?

[https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/resolutions/radiometric](url)

**Comment by semiautomaticgit** [comment, marked as answer]
Hello, the central wavelength defined in SCP is acquired from this link https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/document-library/-/asset_publisher/Wk0TKajiISaR/content/sentinel-2a-spectral-responses that I think it is the most recent update.

However, you can edit the values by changing the code at
https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/blob/1eb6e82bd6c5b9853399b7f9d423c727faa1c313/maininterface/bandsetTab.py#L175
